cmake/InstallTPLs.cmake

```diff
@@ -89,7 +89,7 @@ foreach(_comp ${AXOM_COMPONENTS_ENABLED})
 endforeach()
 
 # TPLs that must be imported
-list(APPEND SPHERAL_EXTERN_LIBS zlib boost eigen qhull silo hdf5 polytope)
+list(APPEND SPHERAL_EXTERN_LIBS boost eigen qhull silo hdf5 polytope)
```

Comment:
We build our own zlib to link to, is this being removed because the system zlib will be used?

Reply:
The zlib path will be included in the CMAKE_PREFIX_PATH in the cmake file created when running TPL. But the zlib target isn't explicitly used when building Spheral.